packages/sources/durin/test/unit/adapter.test.ts

```diff
@@ -0,0 +1,398 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+  process.env.RPC_URL = 'http://localhost:9545'
+
+  describe('validation error', () => {
```

Comment:
Looks like we only have tests for validation errors. Can we add a test that would pass validation and return a 200?

Reply:
I don't think any of our adapters have unit tests that pass validation and return a 200 right now.  These are typically done in the integration tests which we are holding off for now.